backend-demo/src/access.rs

```diff
@@ -0,0 +1,40 @@
+use ashpd::{
+    backend::{
+        access::{AccessImpl, AccessOptions, AccessResponse},
+        request::RequestImpl,
+        Result,
+    },
+    desktop::HandleToken,
+    AppID, WindowIdentifierType,
+};
+use async_trait::async_trait;
+
+#[derive(Default)]
+pub struct Access;
+
+#[async_trait]
+impl RequestImpl for Access {
+    async fn close(&self, token: HandleToken) {
+        tracing::debug!("IN Close(): {token}");
+    }
+}
+
+#[async_trait]
+impl AccessImpl for Access {
+    async fn access_dialog(
+        &self,
+        _handle: HandleToken,
+        _app_id: Option<AppID>,
+        _window_identifier: Option<WindowIdentifierType>,
+        _title: String,
+        _subtitle: String,
+        _body: String,
+        options: AccessOptions,
+    ) -> Result<AccessResponse> {
+        let mut response = <AccessResponse as std::default::Default>::default();
+        for choice in options.choices() {
+            response = response.choice(choice.id(), choice.initial_selection());
+        }
```

Comment:
Do we really want to do this?

Reply:
That's admittedly a dummy implementation that responds with the default (initial) selection. What other behaviour that doesn't involve user interaction would you suggest?